src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -1071,12 +952,8 @@ End Class
                               </Document>)
 
                 state.SendTypeChars(" ")
-                Await state.AssertCompletionSession()
-                Assert.Equal(3, state.CurrentCompletionPresenterSession.CompletionItems.Count)
-                Assert.True(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "b:="))
-                Assert.True(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "num:="))
-                Assert.True(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "str:="))
-                Assert.False(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "dbl:="))
```

Comment:
❕ This assertion was incorrectly translated

Reply:
That it is why the test is disabled. I already fixed it in [https://github.com/dotnet/roslyn/pull/28219](https://github.com/dotnet/roslyn/pull/28219)

Thank you!

---
In reply to: [208003372](https://github.com/dotnet/roslyn/pull/29016#discussion_r208003372) [](ancestors = 208003372)